src/main/java/dukeexecpetions/EventEmptyException.java

```diff
@@ -0,0 +1,26 @@
+package dukeexecpetions;
+
+/** Exception when an empty field is passed for a field*/
+public class EventEmptyException extends EmptyFieldException{
+    /** text for when from entry is missing */
+    private static final String missing_from_txt = "from - What time does %s start from?\n";
+    /** text for when to entry is missing */
+    private static final String missing_to_txt = "to - What time does %s end?\n";
+
+    /**
+     * Checks which field is missing and informs user of missing field
+     *
+     * @param event
+     * @param missing_from
+     * @param missing_to
+     */
+    public EventEmptyException(String event, boolean missing_from, boolean missing_to) {
```

Comment:
Could the variable names use camel case, and reflect their boolean nature?
```suggestion
    public EventEmptyException(String event, boolean isMissingFrom, boolean isMissingTo) {
```